src_c/surface.c

```diff
@@ -1863,6 +1865,52 @@ surf_blit(pgSurfaceObject *self, PyObject *args, PyObject *keywds)
         src_rect = &temp;
     }
 
+    /*
+    Yes, this code is really unprofessional-looking,
+    but there is actually the fastest way known to
+    make this work. Using `PyObject_SetAttr()` on a
+    pgRect in C is actually slower than this method.
+
+    This code is also slower than Python code itself.
+
+    TODO: Optimize or cleanup implementation.
+    */
+    if (anchor_text == NULL) {
+    }
+    else if (strcmp(anchor_text, "topleft") == 0) {
```

Comment:
is it worth some performance if we shorten the anchor names to just to characters? like `topleft = tl`, `bottomright = br`?

Reply:
Thinking more about this, I'm not so sure about this. Any thoughts? @Starbuck5 @ankith26 @illume